matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -207,7 +213,14 @@ impl WebRtcSocket {
     /// Send a packet to the given peer
     pub fn send<T: Into<PeerId>>(&mut self, packet: Packet, id: T) {
```

Comment:
Well if you are open for it it's definitely fine for me - I also believe reliable sending should be the default to avoid confusion for newcomers (like it was for me)